File: adhoc/aneduarte/1765.py
T = int(input())
quantidades = [] 
quantidades.append(T)
geral = []

while T != 0:
    for i in range(0, T, 1):
        numeros = input().split(" ")
        Q = int(numeros[0])
        A = float(numeros[1])
        B = float(numeros[2])
        total = ((A+B)*5/2)*Q
        print("Size #{}:" .format(i+1))
        print("Ice Cream Used: {:.2f} cm2" .format(total))
    print("")
    T = int(input())
    quantidades.append(T)

File: adhoc/lucianoclaudio/1765.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    int t,q;
    double a,b,area;

    cin >> t;
    while(t>0){
        for (int i=1;i<=t;i++){
            cin >> q >> a >> b;
            area = (a+b)*5/2;
            area *= q;
            cout << "Size #" << i << ":\n";
            cout << fixed << setprecision(2);
            cout << "Ice Cream Used: " << area << " cm2\n";
        }
        cout << "\n";
        cin >> t;
    }
    

    return 0;
}